l10n/sled/zh-cn/xml/book_security.xml

```diff
@@ -21,6 +21,24 @@
    </dm:bugtracker>
    <dm:translation>yes</dm:translation>
   </dm:docmanager>
+  <meta name="title" its:translate="yes">安全和强化指南</meta>
+  <meta name="series" its:translate="no">Products &amp; Solutions</meta>
+  <meta name="description" its:translate="yes">如何使用产品随附的安全性软件实现系统安全</meta>
```

Comment:
The same as for SLES: You accidentally picked the wrong line from the spreadsheet, this needs to be the metadata for the 'Security _and Hardening_ Guide'. Will fix with my next commit.